project-management-application-web/src/app/auth/components/login/login.component.html

```diff
@@ -0,0 +1,24 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<div class="container login-container">
+  <div class="row">
+    <div class="col-md-6 login-form-1">
+      <h3>Welcome to PMA</h3>
+      <form>
+        <div class="form-group">
+          <input type="text" class="form-control" placeholder="Your Username *" value=""/>
+        </div>
+        <div class="form-group">
+          <input type="password" class="form-control" placeholder="Your Password *" value=""/>
```

Comment:
you should also add required attribute , not just the visual `*`.  also, maybe a toggle password visibility would be handy